eksrollup/cli.py

```diff
@@ -231,7 +231,19 @@ def update_asgs(asgs, cluster_name):
                         raise Exception('Instance is failing to terminate. Cancelling out.')
 
                     between_nodes_wait = app_config['BETWEEN_NODES_WAIT']
-                    if between_nodes_wait != 0:
+                    between_nodes_wait_pod_regex = app_config['BETWEEN_NODES_WAIT_POD_REGEX']
+                    if between_nodes_wait != 0 and between_nodes_wait_pod_regex:
+                        try:
+                            pods_ready = pods_in_ready_state(between_nodes_wait_pod_regex)
+                            while not pods_ready:
+                                logger.info(f'Waiting for {between_nodes_wait} seconds before continuing checking for pods being ready...')
+                                time.sleep(between_nodes_wait)
+                                pods_ready = pods_in_ready_state(between_nodes_wait_pod_regex)
```

Comment:
This will still sleep `between_nodes_wait` seconds until the selected pods are ready and will do this multiple times. Maybe a while true with a `break` would be more suitable here.

Reply:
Yes, but this implementation cancels the idea of `between_nodes_wait` because there is the possibility that it will wait more than once if the pods are no in ready

Reply:
So what are you suggesting? a static timeout like 5 seconds?

Reply:
Maybe these two options should mutually exclusive and the pods_ready check should have its own hardcoded timeout, something like 20 seconds, much less than between_nodes_wait

Reply:
I`ll implement `BETWEEN_NODES_WAIT_POD_REGEX_WAIT` with a default vaule of 30 seconds.

Reply:
Done by https://github.com/commercetools/eks-rolling-update/pull/1/commits/843eacfec5a6dbfa396f9f3923ee6be94ab6042b